src/OpenLoco/Config.cpp

```diff
@@ -240,6 +242,7 @@ namespace OpenLoco::Config
         node["showFPS"] = _new_config.showFPS;
         node["uncapFPS"] = _new_config.uncapFPS;
         node["displayLockedVehicles"] = _new_config.displayLockedVehicles;
+        node["buildLockedVehicles"] = _new_config.buildLockedVehicles;
```

Comment:
is it normal to have global cheats. In OpenRCT2 we got rid of global cheats and made it tied to the save (but that required waiting until our new save format was completed)

Reply:
In OpenLoco a lot of the boolean cheats are currently global. Has been since the first cheat, disabling vehicle breakdowns. We may need to revisit this when we add multiplayer, though.